docs/corporate-proxy.md

```
@@ -0,0 +1,165 @@
+          "http_proxy": "http://proxy-alt.company.com:8080",
+          "https_proxy": "http://proxy-alt.company.com:8080"
+        }
+      }
+    }
+  }
+}
+```
+
+## Proxy Formats
+
+### HTTP Proxy
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)